src/LibraryViewExtensionWebView2/Views/LibraryView.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<UserControl x:Class="Dynamo.LibraryViewExtensionWebView2.Views.LibraryView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:wv2="clr-namespace:Microsoft.Web.WebView2.Wpf;assembly=Microsoft.Web.WebView2.Wpf"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:Dynamo.LibraryViewExtensionWebView2.Views"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800">
+    <Grid x:Name="mainGrid">
+        <wv2:WebView2 Visibility="Hidden"/>
```

Comment:
Just curious why the initial visibility is Hidden?

Reply:
@QilongTang because otherwise in the Dynamo Start page will show the Library (also in the LibraryViewExtensionMSWebBrowser was hidden by default).

Reply:
Thanks, good to know